tests/baselines/reference/emitBOM.js

```diff
@@ -2,10 +2,18 @@
 // JS and d.ts output should have a BOM but not the sourcemap
 var x;
 
-//// [emitBOM.js]
-ï»¿// JS and d.ts output should have a BOM but not the sourcemap
-var x;
-//# sourceMappingURL=emitBOM.js.map
+tests/cases/compiler/emitBOM.js(1,2): error TS1127: Invalid character.
+tests/cases/compiler/emitBOM.js(1,3): error TS1127: Invalid character.
+
+
+==== tests/cases/compiler/emitBOM.js (2 errors) ====
+    ï»¿// JS and d.ts output should have a BOM but not the sourcemap
+     
+!!! error TS1127: Invalid character.

```

Comment:
@rbuckton you know what's up with the BOM here? I suspect it's actually an issue with the test harness not roundtripping the BOM correctly (maybe by not reading the output string with the correct encoding) and not a parse/emit bug, but I can't be sure.